dashboard/src/components/AppView/AppControls/StatusAwareButton.test.tsx

```diff
@@ -0,0 +1,77 @@
+import { CdsButton } from "@cds/react/button";
+import actions from "actions";
+import { hapi } from "../../../shared/hapi/release";
+
+import * as ReactRedux from "react-redux";
+import { defaultStore, mountWrapper } from "shared/specs/mountWrapper";
+import StatusAwareButton from "./StatusAwareButton";
+import ReactTooltip from "react-tooltip";
+
+let spyOnUseDispatch: jest.SpyInstance;
+const kubeaActions = { ...actions.kube };
+beforeEach(() => {
+  const mockDispatch = jest.fn();
+  spyOnUseDispatch = jest.spyOn(ReactRedux, "useDispatch").mockReturnValue(mockDispatch);
+});
+
+afterEach(() => {
+  actions.kube = { ...kubeaActions };
+  spyOnUseDispatch.mockRestore();
+});
```

Comment:
Do you really need this?  I think you can get rid of it ?

Reply:
i may not need this. I have added it to be consistent with other test files.
but in my scenario, i am not making any backend calls, so i agree this should not be necessary. I will remove it and see that the tests are still passing fine.